contracts/debt-contracts/RepaymentManager.sol

```diff
@@ -191,12 +190,12 @@ contract RepaymentManager is Initializable, IRepaymentManager, ControllerRole {
 
     // @notice reconcile the loans funding status
     function _updateRepaymentStatus() internal {
-        uint _totalDue;
+        uint256 _totalDue;
         uint256 _totalPaid = totalPaid();
 
-        (,,,,uint loanPeriod,,,) = termsContract.getLoanParams();
-        for (uint lp = 0; lp < loanPeriod; lp++) {
-            (,,,uint due) = termsContract.getScheduledPayment(lp+1);
+        (, , , , uint256 loanPeriod, , , ) = termsContract.getLoanParams();
```

Comment:
I'm curious - why does Solhint keep reformatting this? It seems to alternate between leaving spaces and not leaving spaces...